test/dockerized-integration-tests.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-if [ "$DEBUG" == "true" ]
+if [ -v DEBUG ];
```

Comment:
What does this do? I can't find any documentation of "[" or "test" having  "-v" option anywhere.

Reply:
I just found it from searching how to check whether a variable is set in bash. Here's an example from Stack Exchange

https://unix.stackexchange.com/a/212192

Reply:
I tested it manually and the scripts work for me now where before they would crash.